### DISCUSSION TITLE
Range hover selection not showing

### DISCUSSION THREAD

**Discussion by maranyil** [revolist/revogrid#454] [General]
Hello there!
I'm using Revogrid in a Vue2 project and SCSS and tailwind as primary styling languages. Everything works great, but the range selector does not appear. Not when I want to copy values or select multiple rows. (The range function is enabled and works fine).

**Comment by sven42** [comment]
Hello,

same here with Bootstrap 5.

In dom-tree I can find the following element:

`<revogr-temp-range class="" style="" hidden=""></revogr-temp-range>`

The problem seems to be that the [HTML5 [hidden] attribute](https://getbootstrap.com/docs/5.2/content/reboot/#html5-hidden-attribute) is getting normalized by Bootstrap.

For me, the current work around is to reset this css setting for revo-grid:

```
revo-grid [hidden] {
    display: initial !important;
}
```

**Comment by xcchcaptain** [comment]
> 

https://github.com/revolist/revogrid/issues/344#issuecomment-1203984672
thanks ,this worked for me

**Comment by Goston** [comment]
https://github.com/revolist/revogrid/issues/344#issuecomment-1203984672

Thanks, this saved my life!
